nix/shell.nix

```diff
@@ -21,11 +21,13 @@
     packages = with pkgs;
       [
         coreutils
+        pkg-config
         protobuf
-        rustToolchain # The rust toolchain we constructed above
+        rustToolchain
         nodejs
-        clang
```

Comment:
Why is out? I'm pretty sure my builds were failing without C compiler available.